test/tools/runner/hooks/configuration.js

```diff
@@ -100,6 +100,14 @@ const skipBrokenAuthTestBeforeEachHook = function ({ skippedTests } = { skippedT
 };
 
 const testConfigBeforeHook = async function () {
+  // TODO(NODE-5035): Implement OIDC support. Creating the MongoClient will fail
+  // with "MongoInvalidArgumentError: AuthMechanism 'MONGODB-OIDC' not supported"
+  // as is expected until that ticket goes in. Then this condition gets removed.
+  if (MONGODB_URI && !MONGODB_URI.includes('MONGODB-OIDC')) {
+    this.configuration = new TestConfiguration(MONGODB_URI, {});
+    return;
+  }
```

Comment:
The condition should be flipped here, but it does bring to our attention a future problem when we are able to remove this. The print out below is something we'll still want to reach to get environment information, and an `{}` empty context will result in lots of crashes. 
Since all the OIDC work is its own tasks can we skip running mocha all together for the time being? Just an idea, I'm not sure of a better way to globally skip 🤔